usecases/How_to_View_Email_Statistics.md

```diff
@@ -0,0 +1,5 @@
+# How to View Email Statistics
```

Comment:
I'd like to have more parity with the other SendGrid helper libraries. File names should correspond to the appropriate file name like in each of these repos:
https://github.com/sendgrid/sendgrid-python/tree/main/use_cases
https://github.com/sendgrid/sendgrid-nodejs/tree/main/docs/use-cases
https://github.com/sendgrid/sendgrid-go/tree/main/use-cases

For example, this file would be renamed to `email_stats.md`